libbeat/outputs/codecs/json/json.go

```diff
@@ -0,0 +1,54 @@
+package json
```

Comment:
yeah, I used `codecs` as the original the parent package is already named `outputs`. We want a followup github-issue tracking required renaming?